Mathmagician.Tests/FibonacciTest.cs

```diff
@@ -52,6 +52,49 @@ public void FibonacciEnsureListCount()
 
         }
 
+
+        public int fibotest(int a, int b)
+        {
+
+            int temp = a;
+            a = b;
+            b = temp + b;
+            return b;
+        }
+
+        [TestMethod]
+
+        public void FibonacciSequeceTest()
+        {
+
+            // Arrange
+            Fibonacci fiboListSequence = new Fibonacci();
+
+            //Act
+
+            //known fibo pairs
+
+
+            //8 = 5+3
+
+            //13 = 8+5
+
+            //55 = 34+21
+
+            //144 = 89+55
+
+            //317811  121393 + 196418
+
+
+            //Assert
+            Assert.AreEqual(fibotest(5,3), 8);
+            Assert.AreEqual(fibotest(8,5), 13);
+            Assert.AreEqual(fibotest(34,21), 55);
+            Assert.AreEqual(fibotest(89, 55), 144);
+            Assert.AreEqual(fibotest(196418, 121393), 317811);
+        }
```

Comment:
@guthb In this test, you're not even using the `fiboListSequence` instance. This test isn't testing any method on your Fibonacci class. It looks like you're testing `fibotest`. The point of a unit test is to flesh out behaviour/functionality/method on a class, and in this case the `Fibonacci` class.